### DISCUSSION TITLE
I have a website developed in python now i want to use node at same time

### DISCUSSION THREAD

**Discussion by Yogeshsinghbisht** [community/community#60588] [Programming Help]
### Select Topic Area

Question

### Body

I have a website developed in python ([www.journalsinsights.com](https://www.journalsinsights.com/)) now i want to use node at same time, is it possible?

**Comment by Akash1134** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Yogeshsinghbisht  We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category.

You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. 

I've gone ahead and moved this to the correct category ([Programming Help](https://github.com/orgs/community/discussions/categories/programming-help)) for you. Good luck!
